### DISCUSSION TITLE
Version 1.2 Release Candidate 3

### DISCUSSION THREAD

**Discussion by ErinaSugino** [ErinaSugino/Starbound-SBR-Mod#97] [Update Builds]
## Version 1.2 Patchnotes
- Persistent crewmember data
- Player arousal effects and "in heat" mechanics
- NPC sub-genders
- Actor Relayering (with additional layers for ears and tails and offsets per species)
- Complete rework of sextalk
- Fixes for common issues like Anom's Outpost Overhaul, Vulpes Race, Cutebound Avians and forever-kids

## RC3 Notes
- Added flipped head offsets
- Finished Sextalk dialog
- Fixed heat status effects
- Updated some stuff
- Exhaustive testing needed before release. Focus: Pregnancies for Apex and Avian, arousal mechanics, misaligned objects.

### Important Notes
- Due to the complete overhaul, every sextalk altering mod is incompatible now.

<hr /><em>This discussion was created from the release <a href='https://github.com/ErinaSugino/Starbound-SBR-Mod/releases/tag/v1.2_rc3'>Version 1.2 Release Candidate 3</a>.</em>

**Comment by luciodark** [comment]
My Neki's getting arousal debuffs instead of heat debuffs in this build. Double-checked my config, don't think I broke anything myself this time.

**Reply by ErinaSugino** [reply]
That is correct. The name of the parameter was changed to be in-line with the naming of all the other new parameters. Which means the "future-proof" value added to the current public version of the NekiSxB mod is no longer functionaly.

To fix that for yourself, edit the `/species/neki.species.patch` and change the parameter name `sxbUsesHeat` to `sxbUseHeat`.

**Reply by luciodark** [reply]
Gotcha. The changelog file still says 'Uses' in the example, just so you know.
